prov/verbs/src/verbs_cq.c

```diff
@@ -302,7 +302,7 @@ static inline int fi_ibv_poll_outstanding_cq(struct fi_ibv_msg_ep *ep,
 			    (wc.status != IBV_WC_WR_FLUSH_ERR)) {
 				ret = fi_ibv_wc_2_wce(cq, &wc, &wce);
 				if (ret) {
-					wre->ep = NULL;
+					wre->srq = NULL;
```

Comment:
This change should go into a separate commit

Reply:
It should be dropped from this PR